app/routers/event.py

```diff
@@ -260,3 +272,20 @@ def delete_event(event_id: int,
         # if the deletion is successful
     return RedirectResponse(
         url="/calendar", status_code=status.HTTP_200_OK)
+
+
+def get_location_coordinates(
```

Comment:
Prefer this function to be internal.
We want to prevent an external user to be able to use this API on our behalf.